common/dmfsnmp.c

```diff
@@ -148,14 +148,17 @@ print_mib2nut_memory_struct(mib2nut_info_t *self)
 /*END DEBUGGING*/
 
 int load_neon_lib(void){
+	char *neon_libname = get_libname("libneon.so");
 
 	if( lt_dlinit() != 0 ) {
 		fprintf(stderr, "Error initializing lt_init\n");
 		upsdebugx(1, "Error initializing lt_init\n");
 		return 0;
```

Comment:
Seams a, bug... Maybe I wrote the 0 and Arnaud err or viceversa, I dont remember bur i think err is defined as 0 value, then is only a make up issue... and Arnaud is on vacation until 25, lets wait... No?